packages/zowe-explorer/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 
 ### Bug fixes
 
+- Fixed issue where 401 error could occur when opening PDS members after updating credentials. [#3150](https://github.com/zowe/zowe-explorer-vscode/issues/3150)
```

Comment:
How long after updating creds does an error occur? Can we specify this time in the note? Like, is it immediately after? Or w/in the same user session? Or could it potentially happen any time after -- like a month after?

Requesting super minor changes: "Fixed **an** issue where **a** 401 error ... "

Reply:
For the first question, it's within the same user session. How would you suggest to word this?